spec/Dynamic-Validator.md

```diff
@@ -280,11 +280,12 @@ fn reprioritize(candidates: &mut Vec<Candidate>, target: &Address) {
 ```
 
 ### on TermEnd events
-1. Update `term_id` to the current block number and the next term id
+1. Calculate rewards of the previous block and update `intermediate_rewards`
 2. Renew the nomination expiration of the current validators, and reprioritize them.
     * Reprioritization preserves the relative order of the reprioritized validators.
 3. Remove the expired candidates and give back the deposits
 4. Remove the jailed accounts if the current term is greater than `released_at` and give back the deposits
-5. Calculate rewards of the previous block and update `intermediate_rewards`
+5. Jail inactive validators.
 6. Elect validators
     * Store validators in the ascending order
+7. Update `term_id` to the current block number and the next term id
```

Comment:
Seems like some sentences have periods and some don't. The grammar looks fine on everything. I think you can decide either to use periods or not and then merge changes without my approval.